src/App/use-check-updates.ts

```diff
@@ -0,0 +1,27 @@
+import { message } from 'antd';
+import { useEffect, useState } from 'react';
+
+export function useCheckUpdates(): void {
+  const [, setCheckedUpdates] = useState(false);
+  useEffect(() => {
+    message.loading({ content: `Checking for updates...`, key: 'useCheckUpdates', duration: 0 });
+    try {
+      setCheckedUpdates(true);
+      window.checkForUpdates().then((newerVersion) => {
+        if (newerVersion) {
+          message.info({
+            content: `New version ${newerVersion} available! See Help -> Latest Releases`,
```

Comment:
Can we put a clickable link here?

Reply:
> I also tried to make the updates message clickable, with a URL pointing to the releases page, but this somehow isn't trivial. Given small returns from such an addition, I gave up and I'm just showing a plain message.

Reply:
From ant docs I see that `content` can be a ReactNode. Meaning `conent: <>Click <a href="...">here</a> for latest version</>` should work.

Reply:
I tried multiple ways to produce this ReactNode and make whatever it has clickable (and pointing to an external browser) but failed miserably.

What you suggest also doesn't work :(, it shows the HTML tag verbatim

Reply:
Interesting... Nevermind then